common/src/main/java/org/apache/uniffle/common/audit/RpcAuditContext.java

```diff
@@ -18,24 +18,33 @@
 package org.apache.uniffle.common.audit;
 
 import java.io.Closeable;
+import java.util.Optional;
 
 import org.slf4j.Logger;
 
 import org.apache.uniffle.common.rpc.StatusCode;
 
 /** Context for rpc audit logging. */
 public abstract class RpcAuditContext implements Closeable {
```

Comment:
If having performance degression, can we disable this by config? @maobaolong I see some time consuming operations in the hotspot path.

Reply:
Yes, we can disable the rpc audit log by `rss.server.rpc.audit.log.enabled` and `rss.coordinator.rpc.audit.log.enabled`